### ISSUE TITLE
Can it be highlighted, when cursor under a word?

### ISSUE THREAD

**Issue by Rasarts** [richrace/highlight-selected#96]
It would be better for vim mode. Is it possible to do that?

**Comment by cortopy** [comment]
+1

I think this would be great and would bring Atom closer to expected behaviour in IDEs. Could there be an option to highlight on click or double click?

**Comment by davesnx** [comment]
+1

**Comment by richrace** [comment]
Could you please given me an example?

**Comment by davesnx** [comment]
The possibility of change the event that fire the highlight. (right now: double-click)
Can be possible use one-click or simple mouse over. Example, vim.

**Comment by richrace** [comment]
As I don't use Vim, is this the correct behaviour?

In Vim:

- I use command `:set hlsearch`
- I have a line `this is te|st, test test` (where `|` is my cursor)
- I press `*` (by default) and all tree `test` then become highlighted.

And then within Atom, the binding (`*`) would be changeable via the user's keymap file.

**Comment by DontRelaX** [comment]
+1
is it possible to just highlight word under cursor position without depending on any other things (like vim or key binding)?

**Comment by donaldpipowitch** [comment]
It would be great, if this would work for a simple click, not just double click.

**Comment by yakiang** [comment]
+1 for this

**Comment by toxu** [comment]
I don't know if it is a good idea to register a cursor move event. I change the code slightly then it will highlight the word under the cursor. This also works for mouse one click.
![highlight 2](https://cloud.githubusercontent.com/assets/14917833/12111924/1750bad2-b3d4-11e5-9f44-e49a7c0f87f3.gif)

**Comment by jasonmadigan** [comment]
Yeah, would be nice if it highlighted on a single click - something I've grown used to in sublime

**Comment by VoronaPavel** [comment]
@toxu can you share this?

**Comment by vadimhp** [comment]
This is a much needed setting. Proposed look using radio selection buttons:

```
Highlight behavior:
- On word selection
- On single-clicking a word
```

**Comment by nsa** [comment]
Any updates on this issue?
